addons/fleet/views/fleet_vehicle_model_views.xml

```diff
@@ -48,6 +48,10 @@
                             <group string="Engine" invisible="vehicle_type != 'car'">
                                 <group>
                                     <field name="default_fuel_type" required="1"/>
+                                    <label for="vehicle_range"/>
+                                    <div class="o_row" name="vehicle_range">
+                                        <field name="vehicle_range"/><span>km</span>
```

Comment:
maybe use `o_hr_narrow_field` or `o_hr_narrow_field-size` to make the field less imposing :thinking:
(and maybe it should be done to other fields around that are taking full length when the input will never exceed 10 characters)

Reply:
hmm maybe we should do that separately in an improve fleet UX task